src/Internal/Plutip/Spawn.js

```diff
@@ -1,3 +1,11 @@
 const fs = require("fs");
+const path = require("path");
 
 exports._rmdirSync = path => () => fs.rmdirSync(path, { recursive: true });
+
+exports._removeDirs = dirPath => () => {
+  const dirs = fs.readdirSync(dirPath);
```

Comment:
Why don't you remove the top-level dir with `recursive: true`?

Reply:
the problem is that I don't know the name of the test-cluster folder (respec. the six digits after test-cluster) as it's created on the plutip side. 
also, calling _rmdirSync inside Process.onExit still doesn't cleanup when we exit with ctrl+c (contrary to what I wrote yesterday).
so my solution is to just delete everything inside the tmpDir on exiting a child process and if Plutip tests are aborted with ctrl+c it gets cleaned up on the next run. as far as I can see there is nothing inside the tmpDir that has to be conserved?

Reply:
thanks, that's what I was looking for. there is still a problem with `Process.onExit` though. I try to remove the two directories there but if the process is stopped with ctrl+c nothing happens. I tried to use `Process.onSignal SIGINT` without success